### PR TITLE
🔧 migrate b-slider to NeoSlider

### PR DESCRIPTION
**Pull request by roiLeo**
part of #5852, tested on `/rmrk/mint`

- `b-slider` => `NeoSlider`
- vue composition api `BasicSlider`
- fixed `AttributeTagInput` error

Screenshot 2023-06-27 at 14-00-25 KodaDot Low fees and low carbon minting KodaDot - NFT Market Explorer

### CODE REVIEW

File: components/rmrk/Create/SimpleMint.vue [hunk 1]
```diff
@@ -111,7 +111,7 @@
             data-cy="input-batch-address" />
         </NeoField>
         <BasicSlider
-          v-model="distribution"
+          :model-value="distribution"
```
Comment (Jarsen136) [inline components/rmrk/Create/SimpleMint.vue:114]: This change would break something and then the variable `distribution` would not be changed any more.

**Review by Jarsen136 — CHANGES_REQUESTED** [review]

File: components/rmrk/Create/SimpleMint.vue (hunk 1 above)
Comment (roiLeo) [reply components/rmrk/Create/SimpleMint.vue:114]: What solution do you suggest? when using v-model it tells me that I'm missing required prop

Comment (Jarsen136) [reply components/rmrk/Create/SimpleMint.vue:114]: We could try to add props `@change="sliderChangeHandler"` on it, and we could listen to the update event from `BasicSlider ` component.

**Comment by prury** [comment]
took me some time to find the right conditions for the slider to appear 💀

File: components/rmrk/Create/SimpleMint.vue (hunk 1 above)
Comment (roiLeo) [reply components/rmrk/Create/SimpleMint.vue]: removed <BasicSlider />, now it's cleaner and it works

**Comment by roiLeo** [comment]
> took me some time to find the right conditions for the slider to appear 💀 lmk if the distribution itself needs to be tested via minting

I don't even know what "distribution" does

**Comment by prury** [comment]
> > took me some time to find the right conditions for the slider to appear 💀 lmk if the distribution itself needs to be tested via minting
> 
> I don't even know what "distribution" does

haha i think its to supposed to send a specific amount(defined by the slider) of the NFTs you minted to the address list above, but i'm not sure either

**Pull request merged**